setup.py

```diff
@@ -632,6 +632,7 @@ def pkgconfig_win(flag, cut):
                   # Extension("pcl.pcl_grabber", ["pcl/pcl_grabber.pyx", "pcl/grabber_callback.cpp"], language="c++", **ext_args),
                   # debug
                   # gdb_debug=True,
+                  extra_compile_args=ext_args['extra_link_args']
```

Comment:
In "module" variable passed to ext_modules of setup function, 
The link argument is set.

We do double setting, but why is this process necessary?